components/nav/common.js

```diff
@@ -292,7 +292,7 @@ function LogoutObstacle ({ onClose }) {
               await togglePushSubscription().catch(console.error)
             }
 
-            await wallets.resetClient().catch(console.error)
+            removeLocalWallets().catch(console.error)
 
```

Comment:
We should call disconnectVault() here, so if the user logs out from a public computer or some other untrusted setup, it doesn't leave the vault key behind

Reply:
good recs, thanks!